assets/js/graph.js

```diff
@@ -1,5 +1,5 @@
 async function drawGraph(
-  url,
+  _url,
```

Comment:
Is this no longer in use? Can we just remove this argument and remove this from the fn call from `graph.html` as well